### PR TITLE
Adding additional info to existing description for SKU API's $filter param.

### PR DESCRIPTION
**Pull request by avinashakka**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-04-01/skus.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
             "in": "query",
             "required": false,
             "type": "string",
-            "description": "The filter to apply on the operation."
+            "description": "The filter to apply on the operation. Only **location** filter is supported currently."
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-04-01/skus.json:57]: Can you give an example?

Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-04-01/skus.json:57]: Example was already added as part of a previous PR: https://github.com/Azure/azure-rest-api-specs/pull/7167/files

**Review by yungezz — APPROVED** [review]
hi @avinashakka let me know the PR is ready to merge.

**Comment by yungezz** [comment]
hi @avinashakka is the PR ready to merge?

**Comment by avinashakka** [comment]
> hi @avinashakka let me know the PR is ready to merge.

Yes. Waiting for @smotwani to approve.

**Review by smotwani — APPROVED** [review]

**Comment by avinashakka** [comment]
> > hi @avinashakka let me know the PR is ready to merge.
> 
> Yes. Waiting for @smotwani to approve.

@yungezz Please merge the PR. It has been reviewed and approved internally.

**Pull request merged**